constraint/pkg/client/client.go

```diff
@@ -619,10 +623,32 @@ func (c *Client) RemoveData(ctx context.Context, data interface{}) (*types.Respo
 	return resp, &errMap
 }
 
-// Review makes sure the provided object satisfies all stored constraints.
+// Review makes sure the provided object satisfies constraints applicable for specific enforcement points.
 // On error, the responses return value will still be populated so that
 // partial results can be analyzed.
-func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.QueryOpt) (*types.Responses, error) {
+func (c *Client) Review(ctx context.Context, obj interface{}, opts ...reviews.ReviewOpt) (*types.Responses, error) {
+	var eps []string
+	cfg := &reviews.ReviewCfg{}
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
+	if cfg.SourceEP != "" {
+		for _, ep := range c.enforcementPoints {
+			if cfg.SourceEP == ep {
```

Comment:
ensure case sensitivity

Reply:
Storing `sourceEP` and `c.enforcementPoints` in lowercase at initialisation to avoid multiple `ToLower()` calls while checking for values.